coconut/cmd/configuration_import.go

```diff
@@ -33,23 +33,22 @@ var configurationImportCmd = &cobra.Command{
 	Use:   "import <component> <entry> <file_path>",
 	Aliases: []string{"i", "imp"},
 	Example: `coconut conf import <component> <entry> <file_path>
-coconut conf import <component/entry> <file_path>
-coconut conf import <component> <entry> <file_path> -n
-coconut conf import <component/entry> <file_path> --new-component
-coconut conf import <component/entry> <file_path> --format=json
-coconut conf import <component/entry> <file_path>.json
+coconut conf import <component> <entry> <file_path> --new-component
+coconut conf import <component> <entry> <file_path> --format=json
+coconut conf import <component> <entry> <file_path>.json
```

Comment:
We seem to be missing the `<component>/<entry>` bits.

Reply:
They are not missing as the requirement was to have exactly 3 arguments:
```
coconut conf import <component name> <entry name> <file path> [options]
```

Reply:
I see, so `coconut conf import <component/entry> <file_path>.json` never worked even when it was documented?

Reply:
No. It was a mistake when documentation was added and on second look of the requirements it was then removed. 

If we wish to support that we can added so that we are in line with the 
```coconut conf show command```

Reply:
Yes, that would be splendid, and also the final thing before we merge.

Reply:
The feature implementation looks good, could you please just add back the relevant documentation lines for the `<component>/<entry>` syntax? Once this is done we can merge.